src/Template/Element/packages_panel.ctp

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         DebugKit 3.5.2
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
+/**
+ * @type \DebugKit\View\AjaxView $this
+ * @type array $packages
+ */
+?>
+
+<?php if (!empty($packages)): ?>
```

Comment:
Should we roll the current version into this panel, and clean up the toolbar state a bit?